app/models/borrow_policy.rb

```diff
@@ -4,7 +4,7 @@ class BorrowPolicy < ApplicationRecord
   }
 
   validates :name,
-    presence: true, uniqueness: {case_sensitive: false}
+    presence: true, uniqueness: { scope: :library_id, case_sensitive: false }
```

Comment:
This will fail linting:

https://github.com/testdouble/standard#standardrb-the-rules

> Spaces inside blocks, but not hash literals - In Ruby, the { and } characters do a lot of heavy lifting. To visually distinguish hash literals from blocks, Standard enforces that (like arrays), no leading or trailing spaces be added to pad hashes